components/FooterModal.tsx

```diff
@@ -65,33 +66,46 @@ export default function FooterModal({
       >
         {linkBody}
       </FooterLink>
-      <Modal
-        isOpen={isOpen}
-        onClose={handleOnClose}
-        preserveScrollBarGap
-        scrollBehavior="inside"
-        size={["95vw", "95vw", "80vw", "60vw"]}
-      >
-        <ModalOverlay />
-        <ModalContent rounded="lg">
-          <ModalHeader paddingTop={6}>
-            <TextStyledBold fontSize="2xl">{linkBody}</TextStyledBold>
-          </ModalHeader>
-          <ModalCloseButton _focus={{}} />
-          <ModalBody paddingBottom={6}>
-            {modalBody.map((item) => (
-              <Fragment key={item.title}>
-                <TextStyledBold>{item.title}</TextStyledBold>
-                {item.paragraph.map((paragraph) => (
-                  <Fragment key={paragraph}>
-                    <TextStyled>{paragraph}</TextStyled>
-                  </Fragment>
-                ))}
-              </Fragment>
-            ))}
-          </ModalBody>
-        </ModalContent>
-      </Modal>
+      {/* @ts-ignore */}
+      <SlideIn in={isOpen}>
+        {/* @ts-ignore */}
+        {(styles) => (
+          <>
+            <Modal
+              isOpen={true}
+              onClose={handleOnClose}
+              preserveScrollBarGap
+              size={["95vw", "95vw", "80vw", "60vw"]}
+            >
+              <ModalOverlay onClick={handleOnClose} opacity={styles.opacity} />
```

Comment:
I don't think the `onClick` needs to be here after the scrollBehavior was set back to the default "outside"!

Reply:
Fixed!